.travis.yml

```diff
@@ -46,7 +46,11 @@ matrix:
       env: ALLOW_FAILED=1
     - otp_release: 21.3
       elixir: 1.9.4
+      env: ALLOW_FAILED=1
     - otp_release: 22.3
+      elixir: 1.10.3
+      env: ALLOW_FAILED=1
+    - otp_release: 23.0
```

Comment:
@lukaszsamson I'm not sure how we can easily keep the tests running for many different versions since we still depend on built-in docs. Please let me know if you have a better idea.

Reply:
IMO we've 3 options.
1. Leave things as they are - only support fully passing suite on latest supported otp/elixir combination
2. Successively rewrite failing tests to use embedded modules. This can be difficult for erlang, Kernel and stuff. I think embedding erlang code is doable though I don't know if edoc would work. Otherwise we can add a test only dependency with test erlang modules.
3. Mock doc related APIs to return test documentation.

Reply:
Ok, I think we can stick with `#1` for now but let's keep `#2` in mind as a goal.